ui/src/shared/graphs/helpers.js

```diff
@@ -51,24 +51,32 @@ export const multiColumnBarPlotter = e => {
     }
   }
 
-  const barWidth = Math.floor(2.0 / 3 * minSep)
+  const barWidth = Math.max(Math.floor(2.0 / 3.0 * minSep), 1)
```

Comment:
a comment at this line (and in general in `multiColumnBarPlotter`) would be useful to explain what this calculation means. i don't follow it, and so i'm not sure what this change is fixing either, except in the gross sense that `1` should apparently be the minimum width of a bar.